security/iap-demo/README.md

```diff
@@ -0,0 +1,186 @@
+# IAP Demo
+
+[This demo](https://source.cloud.google.com/drebes-play-org-repos-88ed/iap-demo) aims to show the use of Enterprise BeyondCorp features to expose HTTP(S) services running on-prem via Cloud IAP using the [IAP connector](https://github.com/GoogleCloudPlatform/iap-connector) and Cloud VPN HA for connectivity.
+
+## Requirements
+
+### Services
+
+The following services need to be enabled for the project:
```

Comment:
Isn't the list redundant, since service activation will be done via the project module?